app/models/ucas_matched_application.rb

```diff
@@ -1,10 +1,16 @@
 class UCASMatchedApplication
+  RECRUITMENT_CYCLE_YEAR_FOR_UCAS_MATCHES = 2020
```

Comment:
What if we looked at `created_at` on the match, and checked which cycle bucket it fell into? I don't think any new matches should be created after the cycle closes.